lisa/sut_orchestrator/libvirt/ch_platform.py

```diff
@@ -176,6 +176,22 @@ def _create_node_os_disk(
         self, environment: Environment, log: Logger, node: Node
     ) -> None:
         node_context = get_node_context(node)
+        if environment.runbook.nodes_requirement:
+            node_req: List[schema.NodeSpace] = environment.runbook.nodes_requirement
+            for _, node_space in enumerate(node_req):
+                node_runbook: BaseLibvirtNodeSchema = node_space.get_extended_runbook(
+                    self.node_runbook_type(),
+                    type_name=type(self).type_name(),
+                )
+
+                if node_runbook.disk_img_resize_gib:
+                    qemu_tool = self.host_node.tools[QemuImg]
+                    src_file = node_context.os_disk_base_file_path
+                    qemu_tool.resize(
```

Comment:
The resize operation should be done on the `os_disk_file_path` not the base path.

Reply:
Here we are resizing first and then converting.

Reply:
Using os_disk_file_path now.